include/dlaf/init.h

```diff
@@ -38,19 +38,18 @@ struct configuration {
   bool print_config = false;
   std::size_t num_np_gpu_streams_per_thread = 3;
   std::size_t num_hp_gpu_streams_per_thread = 3;
-  std::size_t umpire_host_memory_pool_initial_block_bytes = 1 << 30;
-  std::size_t umpire_host_memory_pool_next_block_bytes = 1 << 30;
+  std::size_t umpire_host_memory_pool_initial_block_bytes = 1 << 16;
+  std::size_t umpire_host_memory_pool_next_block_bytes = 1 << 16;
   std::size_t umpire_host_memory_pool_alignment_bytes = 16;
   double umpire_host_memory_pool_coalescing_free_ratio = 1.0;
   double umpire_host_memory_pool_coalescing_reallocation_ratio = 1.0;
-  std::size_t umpire_device_memory_pool_initial_block_bytes = 1 << 30;
-  std::size_t umpire_device_memory_pool_next_block_bytes = 1 << 30;
+  std::size_t umpire_device_memory_pool_initial_block_bytes = 1 << 16;
+  std::size_t umpire_device_memory_pool_next_block_bytes = 1 << 16;
```

Comment:
Revert these please. If you'd like to discuss changing the defaults open a separate PR and/or issue.

Reply:
These still need to be reverted.